client/wfnews-war/src/main/angular/src/app/components/wildfires-list/wildfires-list.component.ts

```diff
@@ -271,4 +271,23 @@ export class WildFiresListComponent extends CollectionComponent implements OnCha
     this.doSearch()
   }
 
+  isLocationName() {
+    if (this.locationName && this.locationName !== ""){
+      return true
+    }
+  }
+
+  isSearchText() {
+    if (this.searchText && this.searchText !== ""){
+      return true
+    }
+  }
+
+  locationNameUpdated() {
+    if (this.locationName == '') {
+      this.selectedLat=undefined;
```

Comment:
I don't mind if you don't use semi-colons (not required in typescript outside of specific cases) but don't mix them up or it starts to look weird. Also, it's nice to format things so there's spaces between symbols

`this.selectedLong = undefined` is more readable